INSTRUCTION.md

```diff
@@ -0,0 +1,36 @@
+# Instructions
+
+## Running the MySQL Container
+
+1. Build the MySQL Docker image:
+    ```sh
+    docker build -t mysql-local:1.0.0 -f Dockerfile.mysql .
+    ```
+
+2. Run the MySQL container with a volume attached:
+    ```sh
+    docker run -d --name mysql-container -e MYSQL_ROOT_PASSWORD=1234 -e MYSQL_DATABASE=app_db -e MYSQL_USER=app_user -e MYSQL_PASSWORD=1234 -v mysql_data:/var/lib/mysql -p 3306:3306 mysql-local:1.0.0
```

Comment:
Using hardcoded passwords in the command to run the MySQL container is a security risk. Consider using environment variables or Docker secrets to pass the passwords securely.

Reply:
Resolved